app/posts/apply-for-teacher-training/2019-12-18-digital-accessibility-centre-report.md

```diff
@@ -0,0 +1,110 @@
+
+> The standard of accessibility presented throughout the service was very high.
+
+[Full report (PDF)](https://drive.google.com/file/d/1-cFyRAfUMLNVQRvGb0Bfrnv39CjbVkLj/view?usp=sharing)
+
+Screenshots of the service they used:
+* [Candidate](/apply-for-teacher-training/apply-launch)
+* [Provider](/manage-teacher-training-applications/as-launched-26-nov-2019)
+
+## Key failings and their fixes
+
+All WCAG A and AA failings were fixed between 20 December and 17 January.
+
+We have a [Trello ticket for adding automated accessibility testing using aXe](https://trello.com/c/qqNEqL11/834-spike-set-up-axe-or-other-accessibility-testing-tools) which would have prevented many of these issues, and will avoid future regressions.
+
+### Duplicate IDs (WCAG A)
+
+Page 13: When choosing a location for a course, the hint text for each option (which contains the address when present) all had the same ID.
+
+This was [fixed upstream in the govuk_design_system_formbuilder](https://github.com/DFE-Digital/apply-for-postgraduate-teacher-training/pull/1055), and our duplicate IDs were removed when we updated the dependency. ([Trello](https://trello.com/c/LqbMmzbb/692-dac-page-13-fix-duplicate-ids-on-course-site-page))
+
+Automated accessibility testing would highlight this issue.
+
+### Unlabelled form fields (WCAG A)
+
+Page 15: When a training provider gives multiple conditions for an offer, the fields for those conditions don’t have associated labels.
+
+This was [fixed by removing custom IDs from there form fields](https://github.com/DFE-Digital/apply-for-postgraduate-teacher-training/pull/1015). The generated ID then matched the for attribute of the label, meaning that labels were correctly associated. ([Trello](https://trello.com/c/BWbCBEVj/693-dac-page-15-add-form-labels-to-provider-offer-page-conditions))
+
+Automated accessibility testing would highlight this issue.
+
+### Low colour contrast (WCAG AA)
+
+Page 17: The colour contrast of the declined (2.78:1) and offer (3.16:1) statuses shown to providers had insufficient contrast (should be 4.5:1).
+
+This was [fixed by changing our tag styles to use different colours](https://github.com/DFE-Digital/apply-for-postgraduate-teacher-training/pull/1011). ([Trello](https://trello.com/c/VjHpGzot/694-dac-page-17-fix-colour-contrast-of-declined-and-offer-status-on-applications-page))
+
+Many government services create tag styles using white text on the darker colours of the GOV.UK design system palette. Our new colours meet contrast ratios by using pastel colours, for example heavy red text on a light red background. So that other government services can benefit, and so we can stay inline with the design system, we've begun the process of [contributing these designs back upstream to GOV.UK Frontend](https://github.com/alphagov/govuk-frontend/pull/1711).
```

Comment:
Can you say tints/shades instead of pastel?

Reply:
New text:
>  Our new design meets contrast ratios by using tints of those colours